core/quorum_initiator.go

```diff
@@ -656,29 +658,53 @@ func (c *Core) finishConsensus(id string, qt int, p *ipfsport.Peer, status bool,
 	defer c.qlock.Unlock()
 	cs, ok := c.quorumRequest[id]
 	if !ok {
+		fmt.Println("failed to get quorum consensus")
 		if p != nil {
 			p.Close()
 		}
 		return
 	}
+	pd, ok := c.pd[id] //getting details of quorums who pledged
+	if !ok {
+		fmt.Println("failed to get pledged token details")
+		if p != nil {
+			p.Close()
+		}
+		return
+	}
+	keys := make([]string, 0, len(pd.PledgedTokens))
+	for k := range pd.PledgedTokens {
+		keys = append(keys, k)
+	}
+	key := keys[0]
```

Comment:
Can you rename variable `key` into a more clearer name ? something like PledgedTokenDID for eg.  Also is the for loop required ?